dotnet-desktop-guide/framework/wpf/controls/manipulate-columns-and-rows-by-using-columndefinitionscollections.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "How to: Manipulate Columns and Rows by Using ColumnDefinitionsCollections and RowDefinitionsCollections"
+description: Learn how to use the Slider allowing you to select from a range of values by moving a Thumb along a Track.
```

Comment:
I'm seeing the correct article. The description could be something like "Learn how to use methods in the ColumnDefinitionCollection and RowDefinitionCollection classes to perform actions like adding, clearing, or counting contents."